layouts/_default/list.html

```diff
@@ -24,25 +24,25 @@
       {{ $path := split $url.Path "/" }}
       {{ $product := index $path 1 }}
       {{ $version := index $path 2 }}
-      {{ $productVersion := printf "%s/%s" $product $version}}
-      {{ if in .Dir "/v2.x" }}
+      {{ $productVersion := printf "%s/%s" $product $version}} 
+      {{ if or (in .Dir "/v2.0-v2.4") (in .Dir "/v2.5") (in .Dir "/v2.6") }}
       <div class="alert alert-notice">
-          <strong>We are transitioning to versioned documentation.</strong> The v2.x docs will no longer be maintained. <a href="https://rancher.com/docs/rancher/v2.6/en/">Rancher v2.6 docs</a> • <a href="https://rancher.com/docs/rancher/v2.5/en/">Rancher v2.5 docs</a> • <a href="https://rancher.com/docs/rancher/v2.0-v2.4/en/">Rancher v2.0-2.4 docs</a>
-      </div>
+          <strong>You are viewing the documentation in our older website. Rancher Manager docs are in the process of migrating to a new website as we transition from Hugo to the Docusaurus framework. We will update the community with the new site information after it is launched.</strong>
```

Comment:
```suggestion
          <strong>Rancher Manager docs are in the process of migrating to a new website. We will update the community with the new site information after it is launched.</strong>
```

Let's save the `You are viewing the documentation in our older website.` part till we update the banner after launch. As of now this is still our only website for RM docs.

Moving from Hugo to Docusaurus is technical implementation detail which is not really a user concern. However, the content structure change will affect users so that should probably be mentioned.

Reply:
Updated with suggestions.